Helper/ScopeSetting.php

```diff
@@ -143,6 +144,19 @@ public function getWebhookSecret($storeId = null)
         return $this->getScopeSetting(self::WEBHOOK_SECRET, $storeId);
     }
 
+    public function getWebhooks()
+    {
+        $registeredWebhooks = [];
+
+        $product_delete = $this->getProductDeleteBeforeSetting();
+        $product_save = $this->getProductSaveAfterSetting();
+
+        array_push($registeredWebhooks, ['product/delete', $product_delete]);
+        array_push($registeredWebhooks, ['product/save', $product_save]);
```

Comment:
I am seeing [somewhere](https://appdividend.com/2019/03/29/php-array-push-example-add-elements-to-an-array/) that this is possible:
```suggestion
        array_push($registeredWebhooks, ['product/delete', $product_delete]), ['product/save', $product_save]);
```